app/models/concerns/hyrax/user.rb

```diff
@@ -0,0 +1,173 @@
+require 'oauth'
```

Comment:
what was added to this class that required the override?

Reply:
@mcritchlow - These 2 methods are required to override because we don't use password to login.  The batchimporter tool use them.
https://github.com/ucsdlib/horton/blob/feature/ad_ldap/app/models/concerns/hyrax/user.rb#L143
https://github.com/ucsdlib/horton/blob/feature/ad_ldap/app/models/concerns/hyrax/user.rb#L152

Reply:
@VivianChu Could we try to override both methods in models/user.rb instead? I think using something like ` def self.audit_user` and `def self.batch_user` with the same codes should work.

It looks like it could be even better if we can override the class method `def self.find_or_create_system_user(user_key)` instead of those two above in models/user.rb. Could you give it a try? Thanks.

Reply:
@VivianChu - did you have a chance to try @lsitu 's recommendation?